openlibrary/plugins/upstream/account.py

```diff
@@ -805,11 +805,18 @@ def GET(self, username, key='want-to-read'):
                         'author_keys': [a.author.key for a in w.get('authors', [])],
                         'author_names': [a.author.name for a in w.get('authors', [])],
                         'first_publish_year': w.first_publish_year or None,
-                        'lending_edition_s': (w._solr_data.get('lending_edition_s') or
+                        'lending_edition_s': (w._solr_data.get('lending_edition_s') if
```

Comment:
You can also rewrite this as:

```
w._solr_data and w._solr_data.get('lending_edition_s')
```

which may be cleaner and accomplish the same thing